src/deploy/functions/prompts.ts

```diff
@@ -0,0 +1,53 @@
+import * as clc from "cli-color";
+
+import { getFunctionLabel } from "../../functionsDeployHelper";
+import { CloudFunctionTrigger } from "./deploymentPlanner";
+import { FirebaseError } from "../../error";
+import { promptOnce } from "../../prompt";
+import * as utils from "../../utils";
+
```

Comment:
There another prompt for deleting functions that is currently in release.ts - I'll move that here in next PR which refactors release.ts to execute these deploymentPlans.